package.json

```diff
@@ -21,13 +21,13 @@
   "scripts": {
     "test": "jest",
     "test:watch": "jest --watch",
-    "prepublishOnly": "yarn lint && yarn test",
+    "prepublishOnly": "yarn build && chmod +x dist/cli.js && yarn lint && yarn test",
```

Comment:
... `build` wasn't already here!? :grimacing: Whoops! Missed that too. That could have been unfortunate.